resources/views/layouts/default/flights/fleet.blade.php

```diff
@@ -0,0 +1,65 @@
+@extends('app')
+@section('title', __('home.welcome.title'))
```

Comment:
Change the title. This file also needs to be formatted properly like the other templates are formatted - 2 space indents, etc

Reply:
done